lib/committee/middleware/base.rb

```diff
@@ -13,7 +13,7 @@ def initialize(app, options={})
         @raise = options[:raise]
         @schema = self.class.get_schema(options)
 
-        @router = @schema.build_router(options)
+        @router = @schema.build_router(options, is_request: Committee::Middleware::RequestValidation == self.class)
```

Comment:
This code is bad code, we should divide option for RequestValidation and ResponseValidation.
But this code will be removed next major version and this commit doesn't use next version so its ok.